# HG changeset patch
# Parent 11f6a35d8388f439aa73b56ef229338835ac2116

Use FindUser instead of FindClient and FindServer to find a user.

diff -r 11f6a35d8388 ircd/m_check.c
--- a/ircd/m_check.c
+++ b/ircd/m_check.c
@@ -151,12 +151,7 @@
     else
       send_reply(sptr, ERR_SEARCHNOMATCH, "CHECK", parv[1]);
   }
-  else if ((acptr = FindClient(parv[1])) && !(FindServer(parv[1]))) { /* client and not a server */
-    if (!IsRegistered(acptr)) {
-      send_reply(sptr, ERR_SEARCHNOMATCH, "CHECK", parv[1]);
-      return 0;
-    }
-
+  else if ((acptr = FindUser(parv[1]))) {
     checkClient(sptr, acptr);
   }
   else if ((acptr = FindServer(parv[1]))) { /* server */
